### ISSUE TITLE
Contrast theme: Non-hovered menu items in account settings area have nearly invisible text

### ISSUE THREAD

**Issue by sarayourfriend** [Automattic/wp-calypso#49134]
### Steps to reproduce the behavior

1. Switch to the "contrast" theme
2. Go to `https://wordpress.com/me/account`
3. Don't click anything, and note the color of the text of the "Account Settings" nav item on the left-side navbar

#### What I expected to happen
The text should be clearly legible

#### What actually happened
It is nearly invisible

### Context

I'm unsure if this is related to nav unification.

#### Browser / OS version
Firefox, macOS Big Sur

#### Is this specific to the applied theme? Which one?
"contrast"

#### Does this happen on simple or atomic sites or both?
n/a

#### Is there any console output or error text?
n/a

#### Level of impact (Does it block purchases? Does it affect more than just one site?)
Accessibility

#### Screenshot / Video: If applicable, add screenshots to help explain your problem.
<img width="302" alt="Captura de Tela 2021-01-21 às 09 37 53" src="https://user-images.githubusercontent.com/24264157/105389416-ac790380-5bcc-11eb-883a-894ab4ae4aa1.png">
<img width="322" alt="Captura de Tela 2021-01-21 às 09 37 57" src="https://user-images.githubusercontent.com/24264157/105389420-ad119a00-5bcc-11eb-876b-43f18710b755.png">

**Comment by ryelle** [comment]
I think this is a duplicate of https://github.com/Automattic/wp-calypso/issues/48009

**Comment by sarayourfriend** [comment]
@ryelle It is, sorry! Closing.
